library/core/src/intrinsics.rs

```diff
@@ -72,7 +72,7 @@ extern "rust-intrinsic" {
     ///
     /// The stabilized version of this intrinsic is available on the
     /// `std::sync::atomic` types via the `compare_exchange` method by passing
-    /// [`Ordering::SeqCst`](../../std/sync/atomic/enum.Ordering.html)
+    /// [`Ordering::SeqCst`](crate::sync::atomic::Ordering::SeqCst)
```

Comment:
This is a _lot_ of `crate::sync::atomic::Ordering` throughout the file. Maybe it makes sense to `#[allow(unused_imports)] use crate::sync::atomic::Ordering;` at the top with a comment that it's for intra-doc links? Then you could remove these links altogether.

Reply:
Sounds good to me.